service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -14,10 +14,46 @@ To access the most current features of {ProductName}, upgrade to the most curren
 [id="ossm-upgrading-upgrade_{context}"]
 == Upgrading {ProductName}
 
-To upgrade {ProductName}, you must create an instance of {ProductName} 2.0 in a new namespace. Then, once it's configured, move your microservice applications and workloads from your old mesh to the new service mesh.
+To upgrade {ProductName}, you must create an instance of {ProductName} `ServiceMeshControlPlane` v2 resource in a new namespace. Then, once it's configured, move your microservice applications and workloads from your old mesh to the new service mesh.
 
 .Procedure
 
+. Check your v1 `ServiceMeshControlPlane` resource configuration to make sure it is valid. 
++
+.. Run the following command to view your `ServiceMeshControlPlane` resource as a v2 resource.
++
+[source,terminal]
+----
+oc get smcp -o yaml
+----
++
+.. Check the `spec.techPreview.errored.message` field in the output for information about any invalid fields.
++
+.. If there are invalid fields in your v1 resource, you must fix any invalid v1 fields in your v1 resource before you can edit a v2 `ServiceMeshControlPlane` resource. You replace, patch, or edit the v1 resource with the `oc` command.
```

Comment:
How about this:

If there are invalid fields in your v1 resource, the resource will not be reconciled and cannot be edited as a v2 resource (all updates to v2 fields will be overridden by the original v1 settings).  To fix the invalid fields you can replace, patch, or edit the v1 version of the resource.  (Note: you can delete the resource without fixing it.)  After the resource has been fixed, it can once again be reconciled, and you will be able to modify/view the v2 version of the resource.